components/Nav.js

```diff
@@ -8,16 +8,20 @@ import React from 'react'
 
 // Constants
 const links = {
-  Blog: {
-    href: '/blog/all',
-    as: '/blog',
-  },
-  Statistics: {
-    href: '/statistics',
-  },
-  Leaderboard: {
-    href: '/leaderboard',
-  },
+Blog: {
```

Comment:
WHOOPS. Forgot to mention :/

Uhh indentation is extremely important to get correct. a single space wrong and the whole project will refuse to build.
each item within a curly brace should be indented 2 spaces more than it's parent.